cmd/kyma/install/cmd.go

```diff
@@ -102,15 +101,18 @@ func (cmd *command) Run() error {
 		return errors.Wrap(err, "Could not initialize the Kubernetes client. Make sure your kubeconfig is valid")
 	}
 
+	s := cmd.NewStep("Validating configuration")
 	if err := cmd.validateFlags(); err != nil {
+		s.Failure()
 		return err
 	}
+	s.Successf("Configuration validated")
 
-	s := cmd.NewStep("Checking installation source")
+	s = cmd.NewStep("Checking installation source")
 	if cmd.opts.Local {
 		s.LogInfof("Installing Kyma from local path: '%s'", cmd.opts.LocalSrcPath)
 	} else {
-		s.LogInfof("Installing Kyma in version '%s'", cmd.opts.ReleaseVersion)
+		s.LogInfof("Installing Kyma in version '%s'. Config version '%s'", cmd.opts.ReleaseVersion, cmd.opts.ConfigVersion)
```

Comment:
As a user, I wouldn't understand the difference between the installer image version and config version from this message. Either we need to make it more clear what we mean here, or just print the kyma version.

Reply:
Also, when I provide a docker image, this is the output `Installing Kyma in version ''`. It should print the docker image I provided.